src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/RespondentC8Document.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.prl.models.dto.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.prl.models.Element;
+import uk.gov.hmcts.reform.prl.models.complextypes.citizen.documents.ResponseDocuments;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@Builder(toBuilder = true)
```

Comment:
Include @JsonInclude(JsonInclude.Include.NON_NULL)

Reply:
added the above